observation_portal/requestgroups/views.py

```diff
@@ -164,16 +193,32 @@ def get(self, request):
         return Response(pressure.data())
 
 
-class ObservationPortalLastChangedView(APIView):
+class ObservationPortalLastChangedView(APIView, GetSerializerMixin):
     '''
         Returns the datetime of the last status of requests change or new requests addition
     '''
     permission_classes = (IsAdminUser,)
+    schema = ObservationPortalSchema(tags=['RequestGroups'], is_list_view=False)
+    filter_class = LastChangedFilter
+    filter_backends = (DjangoFilterBackend,)
+    serializer_class = import_string(settings.SERIALIZERS['requestgroups']['LastChanged'])
```

Comment:
Same question about if this is still needed with `get_response_serializer` set

Reply:
Nope - is technically redundant. For viewsets I used the `serializer_class` attribute as the "fallback default" in case we didn't define one for an action, but I suppose there's no sense in keeping with that an APIView